### ISSUE TITLE
remove double apt install

### ISSUE THREAD

**Issue by mstormi** [openhab/openhabian#1278]
from openhab.bash as soon as it is no longer required for a fully working install.
First install used to give a "half-installed" result only and required to repeat the command to fully finish installation.

**Comment by mstormi** [comment]
@ecdye Ethan have another close look at the output of the 1st apt install in the migration routine when you test

**Comment by ecdye** [comment]
Will do, I'll try to test later tonight

**Comment by ecdye** [comment]
Seems to be fixed on fresh install for me? what about you?

**Comment by mstormi** [comment]
> 
> 
> Seems to be fixed on fresh install for me? what about you?

I had added duplicated the apt install line in master before opening this issue here to remindus to remove it some day.
So double check what code was your fresh install based on, single or double install ?

**Comment by ecdye** [comment]
double install. I am pretty sure that this is a bug with the dpkg package. Have you reported this to the @BClark09?

**Comment by mstormi** [comment]
Not yet. Can you create an issue in openhab-distro for it ?

BTW: https://github.com/openhab/openhab-distro/issues/1192#issuecomment-735246598

**Comment by BClark09** [comment]
https://github.com/openhab/openhab-linuxpkg/issues/184

I'll take a look this evening.

**Comment by mstormi** [comment]
ok I added the 1197 to not forget about things before I saw your comment here Ben

FWIW I'm right now working on a PR for the restore script so it can migrate configs to OH3 like the postinst script can today.
[ right now I'm wondering if upgrade.lst is persistent in the filesystem or if it's only part of the package. 
In the latter case I would need to duplicate all the code/migration instructions which I'd like to avoid.
